osie-runner/run.py

```diff
@@ -102,6 +138,9 @@ def sanitize_cacher_data(j):
     cmdline_content = cmdline.read()
     tinkerbell = parse.urlparse(util.value_from_kopt(cmdline_content, "tinkerbell"))
     facility = util.value_from_kopt(cmdline_content, "facility")
+    kernel_tp = util.value_from_kopt(cmdline_content, "traceparent") # opentelemetry
```

Comment:
Wondering if maybe we want to shorten this kopt.  I keep going back and forth on it (it's not really a big deal either way) and maybe this comment is better made on the corresponding boots PR.

Reply:
I seriously considered that, but given how new otel is to most folks, let alone those of us in bare metal land, I figured it was better to keep the long name for clarity when people ask "wtf is this?"